lib/modules/datasource/npm/get.spec.ts

```diff
@@ -219,7 +219,7 @@ describe('modules/datasource/npm/get', () => {
     registryUrl = resolveRegistryUrl('error-404');
     expect(await getDependency(http, registryUrl, 'error-404')).toBeNull();
 
-    httpMock.scope('https://test.org').get('/error4').reply(200, null);
+    httpMock.scope('https://test.org').get('/error4').reply(200);
```

Comment:
```suggestion
    // return invalid json to get coverage
    httpMock.scope('https://test.org').get('/error4').reply(200, '{');
```

should be invalid for  coverage to pass

Reply:
```jshttpMock.scope('https://test.org').get('/error4').reply(200, null as never);```
I have been using this in such cases.